doc/GettingStarted.md

```diff
@@ -0,0 +1,259 @@
+# Getting Started with ACON (Attested Containers)
+
+## Building ACON Components
+**Note**: Examples in this section assumes all ACON related files to be installed/stored in `~/acon`, which could be replaced by any directory path deemed suitable by the user.
+
+### Building and Testing the Guest Kernel
+
+You will need a working *aTD* (ACON TD) before launching ACON *Image*s. Booting an *aTD* requires a guest kernel and an initrd image.
+
+1. See [Building Linux Kernel](TDGuest.md#building-linux-kernel) for instructions on configuring and building the guest kernel.
+
+2. Assuming the kernel build directory is `/path/to/linux/build_dir`, the commands blow install the kernel to `~/acon/kernels`.
```

Comment:
typo: blow -> below

Reply:
Fixed in the commit https://github.com/intel/ACON/pull/55/commits/d45a8e99611ccd18e5efe0499a4ded27b5f085bf